extensions/wikia/Recirculation/js/incontent.js

```diff
@@ -0,0 +1,187 @@
+/*global require*/
+require([
+	'jquery',
+	'wikia.window',
+	'wikia.abTest',
+	'wikia.nirvana',
+	'wikia.mustache',
+	'ext.wikia.recirculation.tracker',
+	'ext.wikia.recirculation.utils'
+], function ($, w, abTest, nirvana, Mustache, tracker, utils) {
+	// Currently only showing for English communities
+	if (w.wgContentLanguage !== 'en') { return; }
+
+
+	var $container = $('#mw-content-text'),
+		template;
+
+	function injectInContentWidget($container) {
+		var $links = $container.find('a'),
+			sections = buildSections($container.find('h2')),
+			width,
+			firstSuitableSection,
+			topTitles;
+
+		// If this page doesn't have enough content (either links or sections) we
+		// don't want to show this widget
+		if ($links.length < 8 || sections.length < 3) {
```

Comment:
What do you think about moving those "magic" numbers to some variables which names describe their purpose for example: `minLinksNumber` and `minSectionsNumber`?